queue_services/entity-filer/src/entity_filer/filing_processors/change_of_registration.py

```diff
@@ -89,7 +89,8 @@ def update_parties(business: Business, parties: dict, change_filing_rec: Filing)
     # Create and Update
     for party_info in parties:
         # Create if id not present
-        if not party_info.get('officer').get('id'):
+        if not party_info.get('officer').get('id') or \
+                (party_info.get('officer').get('id') and not isinstance(party_info.get('officer').get('id'), int)):
```

Comment:
I think this is correct: if the FE assigned an id then it's a GUID string. If the BE assigned an UI then it's an integer. Is that the point here? Definitely worth a code comment to explain it.